src/views/Stake/Stake.jsx

```diff
@@ -371,15 +383,25 @@ function Stake() {
 
                   <div className={`stake-user-data`}>
                     <div className="data-row">
-                      <Typography variant="body1">Unstaked Balance</Typography>
                       <Typography variant="body1">
+                        <Trans>Unstaked Balance</Trans>
+                      </Typography>
+                      <Typography variant="body1">
+                        <Trans>Your Balance</Trans>
+                      </Typography>
+                      <Typography variant="body1" id="user-balance">
                         {isAppLoading ? <Skeleton width="80px" /> : <>{trim(ohmBalance, 4)} OHM</>}
                       </Typography>
                     </div>
 
                     <div className="data-row">
-                      <Typography variant="body1">Staked Balance</Typography>
                       <Typography variant="body1">
+                        <Trans>Staked Balance</Trans>
+                      </Typography>
+                      <Typography variant="body1">
+                        <Trans>Your Staked Balance</Trans>
+                      </Typography>
+                      <Typography variant="body1" id="user-staked-balance">
                         {isAppLoading ? <Skeleton width="80px" /> : <>{trimmedBalance} sOHM</>}
                       </Typography>
                     </div>
```

Comment:
lines 411, 414, 420, 423 below need translations. What is the proper way for a dev to notify you?

Reply:
The process would be the following:
- The dev includes t and Trans into their code
- When their code is merged with dev. CI executes lingui:extract and stores the result somewhere (JesusfChrist is evaluating if this is possible)
- A discord bot posts the result in the app-translation discord chat.